mappings/net/minecraft/client/particle/ParticleManager.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS none/bnm net/minecraft/client/particle/ParticleManager
+	FIELD a world Lnone/aiw;
+	FIELD b PARTICLE_TEX Lnone/kp;
+	FIELD c particleQueues [[Ljava/util/ArrayDeque;
+	FIELD e textureManager Lnone/bxf;
+	FIELD f random Ljava/util/Random;
+	FIELD g factoryMap Ljava/util/Map;
+	FIELD h newParticles Ljava/util/Queue;
+	METHOD <init> (Lnone/aiw;Lnone/bxf;)V
+		ARG 0 world
+		ARG 1 textureManager
+	METHOD a tickParticles ()V
+	METHOD a tickParticleGroup (I)V
+		ARG 0 groupId
+	METHOD a createParticle (IDDDDDD[I)Lnone/bnl;
+		ARG 0 particleId
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 velocityX
+		ARG 5 velocityY
+		ARG 6 velocityZ
+		ARG 7 extra
+	METHOD a registerFactory (ILnone/bnn;)V
+		ARG 0 particleId
+		ARG 1 factory
+	METHOD a tickParticleQueue (Ljava/util/Queue;)V
```

Comment:
Comment as per `tick`->`update` but I will NOT overload these into all being called `update`.